app/models/account/billing_address.rb

```diff
@@ -100,6 +100,18 @@ def billing_address
                                      })
   end
 
+  def stripe_billing_address_json
+    stripe_billing_address = {
+      line1: @billing_address.address1,
+      line2: @billing_address.address2,
+      city: @billing_address.city,
+      state: @billing_address.state,
+      postal_code: @billing_address.zip,
+      country: @billing_address.country
+    }.to_json
+    stripe_billing_address
```

Comment:
@Martouta please confirm:
```suggestion
    {
      line1: @billing_address.address1,
      line2: @billing_address.address2,
      city: @billing_address.city,
      state: @billing_address.state,
      postal_code: @billing_address.zip,
      country: @billing_address.country
    }.to_json
```

Reply:
Right @josemigallas 👍🏼 

@damianpm This does **not** belong to the `model`. It should be either in `helpers` or in `presenters`